lib/customers.js

```diff
@@ -117,7 +117,8 @@ async function updateCustomer (id, data, userToken) {
       'us_ssn_override',
       'sanctions_override',
       'front_camera_override',
-      'suspended_until'
+      'suspended_until',
+      'subscriber_info_override'
```

Comment:
We don't really need override for subscriber info. It's not customer submitted data, so there's no point in approving or rejecting it.